Project-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,28 @@
+html {
+  font-size: 62.5%;
+}
+
+body {
+  background: darkgrey;
+}
+
+.main-container {
+  border: 2px solid gray;
+  width: 15%;
+  margin: 15% auto;
+}
+
+.display {
```

Comment:
The display should have a height to keep it static in size with or without input.